src/Arcus.Scripting.ApiManagement/Scripts/Remove-AzApiManagementDefaults.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [string][parameter(Mandatory = $true)] $ResourceGroup,
+    [string][parameter(Mandatory = $true)] $ServiceName
+)
+
+Write-Host "Start removing API Management defaults..."
+$apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroup -ServiceName $ServiceName 
```

Comment:
Let's include some error-handling in case the ApiManagement-instance cannot be found:
`Write-Error 'Specified API Management instance could not be found'`

Reply:
Guess we'll have to implement the same option here, to check for the result an indicate whether the removal was succesfull or not.

Reply:
Yeszz, was planing to.

Reply:
Can you add this or open an issue @stijnmoreels ?

Reply:
Already done.